lib/Parse/Lexer.cpp

```diff
@@ -1204,6 +1206,69 @@ static bool maybeConsumeNewlineEscape(const char *&CurPtr, ssize_t Offset) {
   }
 }
 
+/// diagnoseZeroWidthMatchAndAdvance - Error invisible characters in delimiters.
+/// An invisible character in the middle of a delimiter can be used to extend
+/// the literal beyond what it would appear creating potential security bugs.
+static bool diagnoseZeroWidthMatchAndAdvance(char Target, const char *&CurPtr,
+                                             DiagnosticEngine *Diags) {
+  // TODO: Detect, diagnose and skip over zero-width characters if required.
+  // See https://github.com/apple/swift/pull/17668 for possible implementation.
```

Comment:
@johnno1962 This URL (this PR) doesn't have possible implementation anymore.
Could you file a JIRA issue in https://bugs.swift.org/ . And update the comment?

Reply:
What do you want me to do here? There is already a Jira [SR-8678](
https://bugs.swift.org/browse/SR-8678) for the zero-width problem. Should I raise a new PR to replace the link to the PR with a link to the Jira? The implementation is under this commit:
https://github.com/apple/swift/pull/17668/commits/f0f08e1e86d0c24921fe8ca71901a5ef6bd39811

Reply:
Yeah, please 1) add a link to f0f08e1 in SR-8678, and 2) raise a PR to replace the link.

Reply:
Done: https://github.com/apple/swift/pull/19364.